packages/manager/apps/dedicated/client/app/dedicatedCloud/dashboard/tiles/general-information/general-information.html

```diff
@@ -27,8 +27,85 @@
 
     <oui-tile-definition
         data-term="{{:: 'ovhManagerPccDashboardGeneralInformation_softwareSolution_term' | translate }}"
-        data-description="{{:: $ctrl.bindings.softwareSolution }}"
-    ></oui-tile-definition>
+    >
+        <oui-tile-description>
+            <span data-ng-bind=":: $ctrl.bindings.softwareSolution"></span>
+            <span
+                class="oui-badge oui-badge_warning"
+                data-ng-if="$ctrl.bindings.updateAvailable && !$ctrl.currentService.vcenterUpgradePendingTask"
+                data-translate="ovhManagerPccDashboardGeneralInformation_softwareSolution_update_available"
+            >
+            </span>
+            <span
+                class="oui-badge oui-badge_info"
+                data-ng-if="$ctrl.currentService.vcenterUpgradePendingTask"
+                data-translate="ovhManagerPccDashboardGeneralInformation_softwareSolution_update_planned"
+            >
+            </span>
+        </oui-tile-description>
+        <oui-action-menu
+            data-ng-if="$ctrl.bindings.updateAvailable || $ctrl.currentService.vcenterUpgradePendingTask"
+            data-compact
+            data-placement="end"
+        >
+            <a
+                class="oui-dropdown-option mb-1"
+                role="menuitem"
+                data-ng-if=":: $ctrl.isMinorSolutionUpdateAvailable() && !$ctrl.currentService.vcenterUpgradePendingTask"
+                data-ui-sref="app.dedicatedClouds.dashboard.update({
+                    currentService: $ctrl.currentService,
+                    targetVersion: $ctrl.currentService.version.lastMinor,
+                })"
+                data-track-on="click"
+                data-track-name="dedicated::dedicatedClouds::dashboard::update-minor"
+                data-track-type="action"
+            >
+                <span
+                    data-translate="ovhManagerPccDashboardGeneralInformation_softwareSolution_update_minor"
+                    data-translate-values="{
+                        version: $ctrl.currentService.version.lastMinor.major,
+                        build: $ctrl.currentService.version.lastMinor.build
+                    }"
+                ></span>
+            </a>
+            <a
+                class="oui-dropdown-option mb-1"
+                role="menuitem"
+                data-ng-if=":: $ctrl.isMajorSolutionUpdateAvailable() && !$ctrl.currentService.vcenterUpgradePendingTask"
+                data-ui-sref="app.dedicatedClouds.dashboard.update({
```

Comment:
I feel having a variable in the resolve complicates the code because, it has to be passed in as a parameter in the component, has to be defined in the controller scope etc. I think we do not have an advantage doing that in this case. Please let me know if you think otherwise :)

Reply:
@ganeshkumar1989 I think that we should avoid to strongly bind ui router with our views, therefore avoid `ui-sref` usage, that's why I suggested passing this through a `resolve`
Also if we want to reuse the component we have to declare a `app.dedicatedClouds.dashboard.update` state in the app

Reply:
Thanks for the explanation @marie-j . This has been addressed in the refactoring : https://github.com/ovh/manager/pull/3685/files#diff-31b7abc852a4ca299dcf954d9ca6387860b18ee023427ef621c977d798a0a5b7R70